### PR TITLE
Sync owners from cucushift repo

### PR DESCRIPTION
**Pull request by liangxia**
/cc @wzheng1 @xiuwang @anuragthehatter @zhaozhanqi @xueli181114 @yasun1 @lihongan @duanwei33 

Add you the permission to approve PRs in verification-tests.
You can add/update/remove teammates as co-approver/reviewer once the PR is merged

### CODE REVIEW

**Comment by yasun1** [comment]
Thanks very much. /lgtm

File: features/registry/OWNERS [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
+  - xiuwang
 
 reviewers:
-- zhouying7780
-- pruan-rht
+  - pruan-rht
+  - wewang58
+  - wzheng1
+  - xiuwang
+  - zhouying7780
```
Comment (wzheng1) [inline features/registry/OWNERS:9]: I suppose pruan-rht and zhouying7780 can be removed here to reduce any noise to them : )

Comment (liangxia) [reply features/registry/OWNERS:9]: @wzheng1 Once this PR got merged  you have the power to update that.

Comment (liangxia) [reply features/registry/OWNERS:9]: Let me know if you would like me to update it here, or you would like to try/verify the process later.

Comment (wzheng1) [reply features/registry/OWNERS:9]: If it is convenient for you, feel free to update here directly. Thanks!

Comment (liangxia) [reply features/registry/OWNERS]: @wzheng1 Updated

Comment (wzheng1) [reply features/registry/OWNERS]: Thanks!

**Pull request merged**